compiler/rustc_incremental/src/persist/load.rs

```diff
@@ -160,10 +165,11 @@ pub fn load_dep_graph(sess: &Session) -> DepGraphFuture {
         }
     }
 
+    let nightly_build = sess.is_nightly_build();
```

Comment:
Can be moved higher to avoid calling `sess.is_nightly_build()` above in the same function.